src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,443 @@
+| WithdrawalRate                      | RECOMMENDED           | [number][]    | The rate at which the blood was withdrawn from the subject. The unit of the specified withdrawal rate should be in mL/s.                                                                                                                                      |
+| TubingType                          | RECOMMENDED           | [string][]    | Description of the type of tubing used, ideally including the material and (internal) diameter.                                                                                                                                                               |
+| TubingLength                        | RECOMMENDED           | [number][]    | The length of the blood tubing, from the subject to the detector in the default unit centimeter.                                                                                                                                                              |
+| DispersionConstant                  | RECOMMENDED           | [number][]    | External dispersion time constant resulting from tubing in default unit seconds.                                                                                                                                                                              |
+| Haematocrit                         | RECOMMENDED           | [number][]    | Measured haematocrit, meaning the volume of erythrocytes divided by the volume of whole blood.                                                                                                                                                                |
+| BloodDensity                        | RECOMMENDED           | [number][]    | Measured blood density. Unit of blood density should be in g/mL.                                                                                                                                                                                              |
+
+### Time
+
+If manual blood samples, for example, using a COBRA counter are available,
+then time information as detailed below should be added to the `*recording-manual_blood.tsv`
+and the values below should be added to the `*recording-manual_blood.json`.
+If blood samples using an autosampler, for example, using an Allogg autosampler are available,
+then time information as detailed below should be added to the `*recording-autosampler_blood.tsv`
+and the values below should be added to the `*recording-autosampler_blood.json`.
+
+| **Column name** | **Description**                                                              | **Units**                             |
+| --------------- | ---------------------------------------------------------------------------- | ------------------------------------- |
+| time            | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5) | Unit of time steps (for example, "s") |
```

Comment:
At the top of the PET specification it states "In general, SI units must be used "

So this must be in units of seconds, right? Not "for example in seconds". Imagine someone would specify the values in the table in minutes, would it then still be correctly interpretable?

Reply:
```suggestion
| time            | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5) | Unit of time steps in seconds |
```

Reply:
Agreed, we made the wording more specific here.